db/migrate/20210119225103_create_answers.rb

```diff
@@ -0,0 +1,11 @@
+class CreateAnswers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :answers do |t|
+      t.boolean :correct, null: false
```

Comment:
Вот так ты сможешь гарантировать, что по умолчанию false
```suggestion
      t.boolean :correct, null: false, default: false
```